package/src/steps/FormView/styles.module.css

```diff
@@ -1,7 +1,7 @@
 .terms {
 /*     margin-left: calc( var(--icon-width) / 2); */
     font-size: 0.75rem;
-    padding-top: var(--body-space-childs);
+    padding-top: 28px;
```

Comment:
Should this maybe be a new constant instead of reverting to a magic number? Not sure if the new 28px value is used for other components as well.

Reply:
yes, I created a variable already, but missed adding it here